### PR TITLE
[ONNX] Add test case for onnx::Max scalar type

### PR DESCRIPTION
**Pull request by titaiwangms**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #88751
* #88750

Referenced by minimum cases

### CODE REVIEW

File: test/onnx/test_pytorch_onnx_onnxruntime.py [hunk 1]
```diff
@@ -8728,6 +8728,28 @@ def forward(self, x, y):
         y = torch.full_like(x, True)
         self.run_test(MinimumModel(), (x, y))
 
+    @skipIfUnsupportedMinOpsetVersion(12)
+    def test_maximum_dtypes(self):
+        class MinimumModel(torch.nn.Module):
```
Comment (wschin) [inline test/onnx/test_pytorch_onnx_onnxruntime.py:8733]: I witness an event of copy-and-pasting. lol
```suggestion
        class MaximumModel(torch.nn.Module):
```

Comment (AllenTiTaiWang) [reply test/onnx/test_pytorch_onnx_onnxruntime.py:8733]: :satisfied:

**Review by wschin — APPROVED** [review]
LGTM.

**Review by BowenBao — APPROVED** [review]

**Pull request closed without merging**